event_sale_extra_info/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Events Additional information
+=============================
+
+This module extends the functionality of events to support some additional data
+for registrations.
+
+Added at model level:
+
+- *Commercial partner* field. It's related to the *Partner* field, and shows
+  its corresponding parent company if any.
+
+Added in the BI reports:
+
+- *Commercial partner*.
+- *Ticket type*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Marketing > Events > Registrations > Create*.
+#. You will see the new field.
+
+To perform a BI analysis, you need to:
+
+#. Go to *Reporting > Event > Events Analysis*.
+#. You can use the new fields.
```

Comment:
"clicking on the plus sign in any of the axis of your analysis and selecting the field ..."